packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -393,15 +393,18 @@ export class MaterialPanel extends ConnectedLitElement {
     }
   }
 
-  async onTextureUpload(uri: string, textureInfo: TextureInfo) {
-    if (this.thumbnailsById.has(uri)) {
+  async onTextureUpload(
+      detail, texturePicker: TexturePicker, textureInfo: TextureInfo) {
```

Comment:
What type is detail? Can this be specified?

Reply:
Yeah, I could create a type for it; probably a good idea. Unfortunately I'll still have to cast it, since events are a generic API and `detail: any` is what we get.